src/components/ErrorModal.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
+import {
+  map,
+  split,
+} from 'lodash';
```

Comment:
OOC why are we importing this stuff from lodash? Are the native implementations insufficient for some reason?

Reply:
@zburke, this is a good question. Actually I was thinking that we use `lodash` for performance improvements.